README.md

```diff
@@ -22,7 +22,4 @@ Tutorial will be helpful to you for customizing template-base!)
 
```

Comment:
@daisukenishino 

How about guiding the users, who use Visual Studio 2013 or earlier, to use OpenTouryoTemplate 01-80?

For example:
When using Visual Studio 2013 or earlier, please use *[OpenTouryoTemplates version 01-80](https://github.com/OpenTouryoProject/OpenTouryoTemplates/tree/01-80)*.